lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -565,15 +565,16 @@ def default_route_reflector_client
     # sense to split up the individual methods to support them
     def send_community
       val = config_get('bgp_neighbor_af', 'send_community', @get_args)
-      return default_send_community if val.nil?
+      return default_send_community if val.nil? || val.empty?
       platform == :nexus ? send_community_nexus(val) : send_community_xr(val)
     end
 
     # Nexus: <state> send-community [ both | extended | standard ]
     #  NOTE: 'standard' is default but does not nvgen on some platforms
     #  Returns: none, both, extended, or standard
     def send_community_nexus(val)
-      val = val.split.last
+      return 'both' if val.size > 1
```

Comment:
What are the specific contents of `val` when `size > 1`?  Does it nvgen both `extended` and `standard`?